modules/ppcp-wc-gateway/src/Gateway/OXXO/OXXO.php

```diff
@@ -63,7 +63,15 @@ public function init(): void {
 
 		add_filter(
 			'woocommerce_available_payment_gateways',
-			function ( array $methods ): array {
```

Comment:
hm, that's a strange issue, why would anyone call it with `null`?

Also we use this filter in other places besides oxxo.